src/docs/docs/contributing/staging-production-deployment.md

```diff
@@ -70,6 +70,7 @@ Copy `env.example` in the `autodeployment` directory to `.env` and add the appro
   - repo name (Telescope)
   - deployment type (`staging` or `production`)
   - unsplash client id
+  - GITHUB_TOKEN
```

Comment:
Provide a link to get more details on what this is and how to create.

https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token